--- origsrc/kpartsplugin/src/CMakeLists.txt	2010-05-21 04:10:37.000000000 -0500
+++ src/kpartsplugin/src/CMakeLists.txt	2010-05-21 15:13:43.987665200 -0500
@@ -17,7 +17,7 @@ set( kpartsplugin_LIB_SRCS
 	qtbrowserplugin_x11.cpp
 )
 
-kde4_add_library( kpartsplugin SHARED ${kpartsplugin_LIB_SRCS} )
+kde4_add_library( kpartsplugin MODULE ${kpartsplugin_LIB_SRCS} )
 
 target_link_libraries( kpartsplugin
 	${QT_QTGUI_LIBRARY}
@@ -25,4 +25,4 @@ target_link_libraries( kpartsplugin
 	${KDE4_KPARTS_LIBS}
 )
 
-install(TARGETS kpartsplugin DESTINATION /usr/lib/nsbrowser/plugins/)
+install(TARGETS kpartsplugin DESTINATION /usr/lib/mozilla/plugins/)
